extension/recording-a-log.md

```diff
@@ -14,42 +14,44 @@ Log recording is the core functionality provided by Fiddler Jam. A **Log** in Fi
 
-   ![Start capture screen with the capture options expanded](../images/ext/ext-images/extension-start-capturing-extended.png)
+   ![Start capture screen with the Advanced Options expanded](../images/ext/ext-images/extension-start-capturing-extended.png)
 
-   >important Chrome displays an informational banner: **"Fiddler Jam" started debugging this browser**. Closing the notification or clicking **Cancel** won't stop the capturing session. Use the **Stop Capture** button (see the 5th step) to cancel an ongoing recording.
+   >important Chrome displays an informational banner: **"Fiddler Jam" started debugging this browser**. Closing the notification or clicking **Cancel** won't stop the capturing session. Use the **Stop Capture** button (see the next step) to cancel an ongoing recording.
+   ![Chrome warning banner](../images/ext/ext-images/extension-chrome-notification.png)
 
-4. Click anywhere in the browser tab to hide the Fiddler Jam window and to continue recording. To bring the Fiddler Jam window back, click the extension's icon.
+1. Click  **Stop Capture** to end the recording.
 
-   At any time during recording, you can click **_Capturing…_** to see details about what Fiddler Jam records.
+   ![Stop capturing screen](../images/ext/ext-images/extension-stop-capturing.png)
 
-   ![Ongoing capturing screen](../images/ext/ext-images/extension-link-options-show-sessions.png)
+1. Before submitting the log, you could check the recorded log by clicking on **Capture Successful** line. The **Capture Details** screen will load with a list of the captured sessions, screenshots, console logs, etc. Go back to the previous screen to generate a link and submit the log to the cloud portal.
 
-5. Click  **Stop Capture** to end the recording.
+   ![Stop capturing screen](../images/ext/ext-images/extension-link-options-show-sessions.png)
 
-   ![Stop capturing screen](../images/ext/ext-images/extension-stop-capturing.png)
+1. At this point, the captured traffic is recorded as a log but is not yet saved anywhere. Choose the appropriate [**Sharing Options**]({%slug fj-link-options%}), use the **Get Link** button and proceed with [submitting the recorded log to the JAM portal]({%slug fj-submitting-logs%}#submit-via-link).
 
-   At this point, the captured traffic is recorded as a log but is not yet saved anywhere. Proceed with [submitting the recorded log to the JAM portal]({%slug fj-submitting-logs%}#submit-via-link) or by [saving the log as a file]({%slug fj-submitting-logs%}#save-as-file).
+   ![Generate Link](../images/ext/ext-images/extension-link-options-password.png)
 
 ## Additional Recording Options
 
-During log recording, Fiddler Jam can capture not only the traffic but also:
+During log recording, Fiddler Jam could capture not only the HTTP/HTTPS traffic but also:
```

Comment:
`could` is overused and implies uncertainty. Prefer `can` as a simpler word.

```suggestion
During recording, Fiddler Jam monitors the HTTP/HTTPS traffic, but it can also capture:
```